File: dbpro/src/main/java/kr/ac/sejong/feat/actor/MovieActorDTO.java
package kr.ac.sejong.feat.actor;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import kr.ac.sejong.config.ColumnName;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MovieActorDTO {

	@ColumnName("ACTOR_CODE")
	private String actorCode;

	@ColumnName("MOVIE_CODE")
	private String movieCode;

	@ColumnName("ACTOR_ROLE")
	private String actorRole;

}

File: dbpro/src/main/java/kr/ac/sejong/feat/actor/MovieActorDTO.kt
package kr.ac.sejong.feat.actor

import kr.ac.sejong.config.ColumnName

data class MovieActorDTO(
    @ColumnName("ACTOR_CODE")
    private val actorCode: String? = null,

    @ColumnName("MOVIE_CODE")
    private val movieCode: String? = null,

    @ColumnName("ACTOR_ROLE")
    private val actorRole: String? = null
)